frame/support/procedural/src/pallet/expand/storage.rs

```diff
@@ -378,6 +436,13 @@ pub fn expand_storages(def: &mut Def) -> proc_macro2::TokenStream {
 
 			let cfg_attrs = &storage.cfg_attrs;
 
+			// If the storage item is public, just link to it, rather than
```

Comment:
rather than... I guess embed it? 

```suggestion
			// If the storage item is public, just link to it, rather than embed it.
```